wasm/index.go

```diff
@@ -82,7 +82,17 @@ func (m *Module) populateFunctions() error {
 	}
 
 	// Add the functions from the wasm itself to the function list
-	numImports := len(m.FunctionIndexSpace)
+
+	// when imported module not provided, len(m.FunctionIndecSpace) == 0
```

Comment:
could you reword a bit this comment?
I am not sure I completely understand it.

do you mean:
```
// when an imported module is not provided, the functions index space is empty.
// to maintain indices consistency, we compute the number of expected imported functions.
```

Reply:
yes, that's it.